addons/mail/static/tests/qunit_suite_tests/components/composer_suggestion_canned_response_tests.js

```diff
@@ -13,23 +13,25 @@ QUnit.module('composer_suggestion_canned_response_tests.js', {
 QUnit.test('canned response suggestion displayed', async function (assert) {
     assert.expect(1);
 
-    this.data['mail.channel'].records.push({ id: 20 });
-    const { createComposerSuggestionComponent, messaging } = await start({ data: this.data });
-    const thread = messaging.models['Thread'].findFromIdentifyingData({
-        id: 20,
-        model: 'mail.channel',
-    });
-    const cannedResponse = messaging.models['CannedResponse'].create({
-        id: 7,
+    const pyEnv = await startServer();
+    const channelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.shortcode'].create({
         source: 'hello',
         substitution: "Hello, how are you?",
     });
-    await createComposerSuggestionComponent(thread.composer, {
-        isActive: true,
-        modelName: 'CannedResponse',
-        recordLocalId: cannedResponse.localId,
+    const { afterNextRender, openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: channelId,
+            },
+        },
+        hasDiscuss: true,
```

Comment:
deprecated, I think it's useless with `openDiscuss`

Reply:
Unfortunately still needed for making `openDiscuss` work